robot_info: PR2

dmp_joint_names: 
  - r_upper_arm_roll_joint_th
  - r_shoulder_pan_joint_th
  - r_shoulder_lift_joint_th
  - r_forearm_roll_joint_th
  - r_elbow_flex_joint_th
  - r_wrist_flex_joint_th
  - r_wrist_roll_joint_th